area/area_blackbox_test.go

```diff
@@ -1,22 +1,20 @@
 package area_test
 
 import (
+	"context"
 	"strconv"
 	"testing"
 	"time"
 
-	"context"
-
 	"github.com/almighty/almighty-core/area"
+	localerror "github.com/almighty/almighty-core/errors"
 	"github.com/almighty/almighty-core/gormsupport/cleaner"
 	"github.com/almighty/almighty-core/gormtestsupport"
 	"github.com/almighty/almighty-core/path"
-	"github.com/pkg/errors"
-
-	localerror "github.com/almighty/almighty-core/errors"
 	"github.com/almighty/almighty-core/resource"
 	"github.com/almighty/almighty-core/space"
 
+	"github.com/pkg/errors"
```

Comment:
https://github.com/almighty/almighty-core/pull/1389/commits/960ba3deddb447e3b51d11c251d1ee00c34799c3Fixed in